en/all.json

```diff
@@ -165,21 +165,5 @@
   "POWERUPS": {
     "POWERUP_one": "Powerup",
     "POWERUP_other": "Powerups"
-  },
```

Comment:
These look like they were mistakenly deleted; please add them back. Thank you!

Reply:
I think this has to be an issue with the Inlang Editor itself, I'll copy over the changes manually any close the PR. Will see if it happens again